HTseq/featureCount.nf

```diff
@@ -0,0 +1,25 @@
+process featureCount {
+    tag "featureCount"
+    publishDir "$params.outdir/${sample}/counts", mode: 'copy'
+    cpus 4
+    penv 'threaded'
+    memory '20 GB'
+    time '1h'
+
+    input:
+    set val(sample), file(bam)
+    
+	
+    output:
+    set val(sample), file("${sample}_read_counts_raw.txt") 
+
+    shell:
+    """
+    set -o pipefail
+    module load python/2.7.10  
```

Comment:
We should probably not use the module system. It is hpc and sometimes even group/user specific. We should move to containers or if that is not possible just use a path variable.